l10n/css.json

```
@@ -759,7 +790,8 @@
     "en-US": "floats",
     "fr": "flottants",
     "ja": "浮動要素",
-    "ru": "плавает"
+    "ru": "плавает",
+    "zh-CN": "浮动"
```

context: https://developer.mozilla.org/zh-CN/docs/Web/CSS/shape-image-threshold

```suggestion
    "zh-CN": "浮动元素"
```